code/modules/mob/living/carbon/human/species_types/rage.dm

```diff
@@ -0,0 +1,56 @@
+/*Result of the rage serum, turning players into what is essentially a walking tank that exists solely to smash those weaker than them. Extremely durable
+but essentially loses their ability to interact in any meaningful way that doesn't involve slamming their fists against it. */
+
+/datum/species/rage
+	name = "Walking Mass of Muscle and Testosterone."
+	id = "buff_hulk"
+	say_mod = "grunts"
+	blacklisted = TRUE
+	default_color = "34AF10"
+	fixed_mut_color = "34AF10"
+	species_traits = list(MUTCOLORS,EYECOLOR,HAIR,FACEHAIR,LIPS,PIERCEIMMUNE,NODISMEMBER,RADIMMUNE)
+	use_skintones = 0
+	mutant_bodyparts = list("tail_human", "ears", "wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	siemens_coeff = 0
+	limbs_id = "human"
+	no_equip = list(slot_head, slot_gloves, slot_wear_suit) //keeps user from wearing most types of armor to prevent additional resistance stacking.
+	armor = 45
+	speedmod = 0
+	brutemod = 0.25
+	burnmod = 0.35
+	heatmod = 1.2
+	stunmod = 0.15
+
+/datum/species/rage/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
+	. = ..()
+	if(H.has_dna())
+		if(H.dna.check_mutation(HULK)) //hulk along with this would be hilariously overpowered, probably
+			H.visible_message("<span class='danger'>[H]'s body twists and contorts violently as their own muscle-mass caves in on them!</span>", \
+							 "<span class='userdanger'>You cry out in agony as you feel your body tighten and concort inward on itself,\
+				 			 the immense amount of muscle practically tearing you apart! Your body can't handle this much power!</span>")
+			H.say("AAAAAARGGHH!!!")
+			H.gib()
+			return
+	for(var/V in H.held_items)
+		var/obj/item/I = V
+		if(istype(I))
+			if(H.dropItemToGround(I))
+				var/obj/item/weapon/melee/buff_arm/h = new /obj/item/weapon/melee/buff_arm()
+				H.put_in_hands(h)
+		else
+			var/obj/item/weapon/melee/buff_arm/h = new /obj/item/weapon/melee/buff_arm()
+			H.put_in_hands(h)
+	H.visible_message("<span class='danger'>[H]'s arms rapidly expand and contort into throbbing masses of muscle, their faces contorting into that of some wild, bloodlusting beast!</span>")
+	H.mind.AddSpell(new /obj/effect/proc_holder/spell/aimed/groundpound)
```

Comment:
1. Add the spell directly to the mob, or you'll have the target autocloning/mind transferring/whatever and keeping the spell. 
2. Ideally you also want to store the spell instance in a var so you can remove it on_species_loss, but given it should gib on loss it might not matter. You never know if there's an edge case, though.